src/cli/commands/format.rs

```diff
@@ -56,7 +56,7 @@ impl ColorCommand for FormatCommand {
                     .paint(output, color.text_color().ansi_style().on(color))
             )?;
         } else {
-            write!(out.handle, "{}", output)?;
+            writeln!(out.handle, "{}", output)?;
```

Comment:
why has this been changed?

Reply:
This change matches the output pattern from the main colorized/if-branch for non-colorized output.

More specifically, the main if-branch is executing a `writeln!` for colorized output. The else-branch should also have it's output end with a newline as well for non-colorized output.

The else-branch *is* triggered on any machine without VT100 color support (early Win10 [and earlier] machines) and it triggers a failing test on those machines.

This fixes it.

Reply:
I looked at it again this evening, and I think the issue *was* a problem with the initial implementation that I submitted. Specifically, I gated the `write_colored_line` clause with `config.color_output` (see [code lines](https://github.com/sharkdp/pastel/commit/92de415c61d3499c5e9b746dac1dec240a28720c#diff-503776d2c929263e2847ddf929c4fa6eR49-R51)).

Now, with the changed implementation, modifying only the flag status, the test failure is no longer present.

I'll drop that commit.